radio/src/gui/480x272/theme.h

```diff
@@ -26,7 +26,7 @@ class BitmapBuffer;
 #define MAX_THEME_OPTIONS              5
 
 class Theme;
-void registerTheme(Theme * theme);
+extern void registerTheme(Theme * theme);
```

Comment:
OK... I've seen it both ways in the OTx code, it seems.  I'm with the 2nd answer on SO, reason # 3. :-D